src/blog/jspi.md

```diff
@@ -0,0 +1,561 @@
+asynchronous computations: the Promise and the `async` function
+notation. The JSPI is designed to integrate well with this but not to
+replace it.
+
+### Next steps
+
+The JSPI is currently experimental–one should not use it for Web
+applications that are intended to be deployed in production. However,
+it is a _standard track_ effort; which means that it will eventually
+become a standard and we expect this to become a standard implemented
+across all major browsers.
+
+The rest of this post focuses on how to access the JSPI, how to
+develop code that uses it and some examples to try out.
+
+## How can I use it today?
+
+JSPI is being developed on Intel x64 and on ARM 64 architectures. It
+is available for Linux, macOS, Windows and ChromeOS. To test it
+locally, go to chrome://flags in Chrome, search for “Experimental
+WebAssembly JavaScript Promise Integration (JSPI)” and check the
+box. Relaunch as suggested for it to take effect.
+
+You should use at least version `110.0.5473.0` (MacOS) / `110.0.5469.0`
+(Windows,Android) / `110.0.5478.4` (Linux) or ChromeOS to get the latest
+version of the API. We recommend using the Canary channel to ensure
+that any stability updates are applied. In addition, if you wish to
+use Emscripten to generate WebAssembly (which we recommend), you
+should use a version that is at least `3.1.28`.
+
+It is not yet possible to enable the feature for end users, only to
+test it locally by enabling this flag. Eventually we hope to do an
+Origin Trial to enable this feature for origins that want to opt in.
+
+Once the flag is enabled, you should be able to run scripts that use
+JSPI. Below we show how you can use Emscripten to generate a
+WebAssembly module in C/C++ that uses JSPI. If your application
+involves a different language, not using Emscripten for example, then
+we suggest that you look at how the API works you should look at the
+[proposal](https://github.com/WebAssembly/js-promise-integration/blob/main/proposals/js-promise-integration/Overview.md).
+
+### Limitations
+
+The chrome implementation of JSPI should already support typical use
+cases. However it is experimental so there are a few limitations to be
+aware of:
+
+
+* Only x64 and arm64 are supported.
+* Only JS-to-wasm exports and wasm-to-JS imports are supported. It is
+  not possible for instance to call a re-imported wasm export and
+  suspend when it returns a Promise.
+* Each call to a JSPI export runs on a separate stack. For now these
+  stacks have a fixed size. The default stack size in kB can be
+  changed with the V8 flag:
+
+  ```
+  –wasm-stack-switching-stack-size
+  ```
+  From the chrome command line, this would look like:
+  ```
+  --js-flags=”--wasm-stack-switching-stack-size=1000”
+  ```
+
+## A small demo
+
+To see all this working, let’s try a simple example. This C program
+computes fibonacci in a spectacularly bad way: by asking JavaScript to
+do the addition, even worse by using JavaScript `Promise`s to do it:[^2]
```

Comment:
```suggestion
do the addition, even worse by using JavaScript Promises to do it:[^2]
```
The formatting seems off with `Promise`s, also inconsistent with the rest of the file that uses Promise/Promises in plain text.